datadistribution.sh

```diff
@@ -0,0 +1,67 @@
+package: DataDistribution
+version: "%(tag_basename)s"
+tag: v0.5.1
+requires:
+  - boost
+  - FairLogger
+  - FairMQ
+  - Monitoring
+  - O2
+build_requires:
+  - CMake
+  - "GCC-Toolchain:(?!osx)"
+  - ms_gsl
+source: https://github.com/AliceO2Group/DataDistribution
+incremental_recipe: |
+  make ${JOBS:+-j$JOBS} install
+  mkdir -p $INSTALLROOT/etc/modulefiles && rsync -a --delete etc/modulefiles/ $INSTALLROOT/etc/modulefiles
+---
+#!/bin/bash -ex
+
+case $ARCHITECTURE in
+    osx*) [[ ! $BOOST_ROOT ]] && BOOST_ROOT=$(brew --prefix boost);;
+esac
+
+cmake $SOURCEDIR                                              \
+      -DCMAKE_INSTALL_PREFIX=$INSTALLROOT                     \
+      ${BOOST_ROOT:+-DBOOST_ROOT=$BOOST_ROOT}                 \
+      ${FAIRLOGGER_ROOT:+-DFairLogger_ROOT=$FAIRLOGGER_ROOT}  \
+      ${FAIRMQ_ROOT:+-DFairMQ_ROOT=$FAIRMQ_ROOT}              \
+      ${O2_ROOT:+-DO2_ROOT=$O2_ROOT}                          \
+      ${MONITORING_ROOT:+-DMonitoring_ROOT=$MONITORING_ROOT}  \
+      -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
+
+cp ${BUILDDIR}/compile_commands.json ${INSTALLROOT}
+make ${JOBS+-j $JOBS} install
```

Comment:
While we are at it: can you use a generic `cmake` command in case we use a different generator (like Ninja)?

This would mean to add:

```
cmake ... ${CMAKE_GENERATOR:+-G "$CMAKE_GENERATOR"} ...
```

to the cmake list of options, and then replacing the `make install` line with:

```
cmake --build . -- ${JOBS+-j $JOBS} install
```

Reply:
OK. updated